apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/pipeline/impl/global_nn_recognizer_crh.hpp

```diff
@@ -377,7 +379,7 @@ template<template<class > class Distance, typename PointInT, typename FeatureT>
         boost::shared_ptr < std::vector<Eigen::Matrix4f, Eigen::aligned_allocator<Eigen::Matrix4f> > > transforms_temp;
 
         models_temp.reset (new std::vector<ModelT>);
```

Comment:
You can use std::make_shared here.

Reply:
Is replacement of every `reset` with `make_shared` our goal? Personally, I see `reset`-based code as more maintainable in a sense that it does not need to know whether the pointer is `std` or `boost`.